Makefile

```diff
@@ -1,21 +1,35 @@
-KICKSTART_FILE=centos-7-minimal.ks
-KICKSTART_TEMPLATE=centos-7-minimal.template
+CENTOS_KICKSTART_FILE=centos-7-minimal.ks
```

Comment:
As said, I don't think all these definitions are needed in this case, given that we use target specific variables below. One might as well change the values there directly. But I can go either way.